### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by briancohan**
Cleaned up docstrings and some minor tweaks.

### CODE REVIEW

**Review by VagishVela — DISMISSED** [review]
I'm getting this weird think for all the code boxes:
Screen Shot 2021-07-16 at 10 35 47 PM

**Comment by briancohan** [comment]
Good catch, I need to textwrap.dedent

File: wtpython/display.py [hunk 1]
```diff
@@ -41,45 +41,51 @@ def store_results_in_module(
...
     ) -> None:
         if questions is not None:
             self.questions = questions
         super().__init__(name=name)
 
     def get_questions(self) -> str:
-        """Put questions into legible format"""
+        """Format question list."""
         text = ""
         for i, question in enumerate(self.questions):
```
Comment (mustafaquraish) [inline wtpython/display.py:74]: Perhaps also use `enumerate(..., start=1)` here, like you did in the previous commit?

**Comment by mustafaquraish** [comment]
Other than that seems mostly fine to me. Some well needed changes. Not sure why you reverted the `pre` tag from a multi-line comment? That seemed much nicer to me. If you were having issues with getting it to render you should try wrapping the string in `textwrap.dedent()`.

File: wtpython/display.py (hunk 1 above)
Comment (briancohan) [reply wtpython/display.py:74]: I fixed all of them, however here there's a balance of comparing to `self.index` and having a 1-indexed presentation format

Comment (briancohan) [reply wtpython/display.py:74]: > 
> 
> Other than that seems mostly fine to me. Some well needed changes. Not sure why you reverted the `pre` tag from a multi-line comment? That seemed much nicer to me. If you were having issues with getting it to render you should try wrapping the string in `textwrap.dedent()`.

Yeah, I tried dedent and still had some issues, so I just put it back to how you had it.

**Review by mustafaquraish — APPROVED** [review]

**Pull request merged**